.github/workflows/ansible-test.yml

```diff
@@ -34,6 +34,48 @@ jobs:
         name: collection
         path: .cache/collection-tarballs
 
+  importer:
+    name: Galaxy-importer check
+    needs:
+    - build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ansible: [2.9.12, 2.10.0]
+    steps:
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install ansible-base (v${{ matrix.ansible }})
+      run: pip install https://github.com/ansible/ansible/archive/v${{ matrix.ansible }}.tar.gz --disable-pip-version-check
+
+    - name: Download migrated collection artifacts
+      uses: actions/download-artifact@v1
+      with:
+        name: collection
+        path: .cache/collection-tarballs
+
+    - name: Install the collection tarball
+      run: ansible-galaxy collection install .cache/collection-tarballs/*.tar.gz
+
+    - name: Install galaxy-importer
+      run: pip install galaxy-importer
+
+    - name: Create galaxy-importer directory
+      run: sudo mkdir -p /etc/galaxy-importer
+
+    - name: Create galaxy-importer.cfg
+      run: sudo cp /home/runner/.ansible/collections/ansible_collections/cisco/aci/.github/workflows/galaxy-importer.cfg /etc/galaxy-importer/galaxy-importer.cfg
+
+    - name: Run galaxy-importer check
+      run: python -m galaxy_importer.main /home/runner/.ansible/collections/ansible_collections/cisco/aci/cisco-aci-*
+
+    - name: Print galaxy-importer check result
+      run: cat importer_result.json
+      working-directory: /home/runner/.ansible/collections/ansible_collections/cisco/aci
```

Comment:
We might want to register it as another artifact look at the build step for it. This will help in case we need to export it or link it to an issue.